internal/cmd/common.go

```diff
@@ -44,7 +44,14 @@ func getProject() (*project.Project, error) {
 
 	if fFileMode {
 		var err error
-		proj, err = project.NewFileProject(filepath.Join(fChdir, fFileName), opts...)
+
+		filePath := filepath.Join(fChdir, fFileName)
+		_, err = os.Stat(fFileName)
+		if filepath.IsAbs(fFileName) && !os.IsNotExist(err) {
```

Comment:
Question: It's a bit odd that we will continue if `err != nil`. I guess the intention was to fallback to `NewFileProject`?

Reply:
Yes, correct. I'll add a comment here.